_data/changelogs/component-footer.yml

```
@@ -2,6 +2,13 @@ title: Footer
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Resolved style specificity issues between `usa-footer` and `usa-layout-grid`.
+    summaryAdditional:  This resolves visual regressions potentially caused by removing layout grid dependency.
```

Tried a summary that was a bit more explicit and plain language. Does this capture the issue?
```suggestion
    summary: Fixed a bug that caused some grid utility classes to be ignored when used inside `usa-footer`.
    summaryAdditional:  
```

---

Yeah, I think that generalizes it well. Should we mention that it resolves a visual regression since some users have noticed? Or do you think it's unnecessary?

---

If the visual regression is caused by the grid utility classes being ignored, then I think it is covered. Short and sweet is good here, I think. Open to other input though!

---

Sounds good to me! Committed 👍